cmd/machine-config-daemon/start.go

```diff
@@ -52,7 +51,7 @@ func runStartCmd(cmd *cobra.Command, args []string) {
 	// To help debugging, immediately log version
 	glog.Infof("Version: %+v", version.Version)
 
-	operatingSystem, err := osrelease.GetHostRunningOS(startOpts.rootMount)
+	operatingSystem, err := daemon.GetHostRunningOS(startOpts.rootMount)
```

Comment:
Looks like this hunk is just nullifying an opposite hunk in the previous commit. Might as well untangle those to make the commits cleaner.

Reply:
re pushed cleaning up all this mess and actually moved osrelease back to daemon cause, meh, we don't really need it outside except cmd/ but that's gonna be cgo anyway